### ISSUE TITLE
[Bug] RomM gets stuck while triying to scan some roms (.7z extension related)

### ISSUE THREAD

**Issue by outerwinnie** [rommapp/romm#1211]
**RomM version**
3.5.1

**Describe the bug**
It seems that if a rom have .7z extension and goes over certain size, RomM gets permanently stuck scanning that one rom.

**To Reproduce**

1. Scan a rom file with .7z and over 4Gb (maybe 3Gb)
2. See error

**Expected behavior**
It shouldn't get stuck there, and should identify the rom correctly as (i think) was pre 3.5.0

**Comment by gantoine** [comment]
Can you post the scan logs?

**Comment by outerwinnie** [comment]
> Can you post the scan logs?

Sorry, I post it on Discord and forgot to post it here to, here it is:

```
`INFO:      [RomM][2024-09-21 20:45:38] Found 1 platforms in file system 
INFO:      [RomM][2024-09-21 20:45:38] · PC
20:45:37 high: endpoints.sockets.scan.scan_platforms([4], <ScanType.QUICK: 'quick'>, [], ['igdb']) (e29178f2-5a99-4d52-b92e-cb16e7eba171)
INFO:      [RomM][2024-09-21 20:45:37] 🔎 Scanning 
INFO:      [RomM][2024-09-21 20:45:39]   Identified as PC (Microsoft Windows) 🎮
WARNING:  [RomM][2024-09-21 20:45:39]   ⚠️ No firmware found, skipping firmware scan for this platform
INFO:      [RomM][2024-09-21 20:45:39]   22 roms found
INFO:      [RomM][2024-09-21 20:45:39]      · Silent Hill 3.7z
20:46:57 Moving job to FailedJobRegistry (Work-horse terminated unexpectedly; waitpid returned 9 (signal 9); )`
```

**Comment by adamantike** [comment]
@rarrais, the issue with `.tar.gz` files is a different one, but we were able to come up with a fix for it in #1260.
